Rectangle/Snapping/SnappingManager.swift

```diff
@@ -419,6 +440,32 @@ class SnappingManager {
                 }
                 return SnapArea(screen: screen, action: .firstThird)
             }
+            
+            // check second quarter column
+            if Defaults.quarterColumnsSnapAera.userEnabled {
+                if loc.x >= frame.minX + quarterWidth && loc.x <= frame.maxX - quarterWidth*2{
+                    if let priorAction = priorSnapArea?.action {
+                        let action: WindowAction
+                        switch priorAction {
+                        case .firstFourth, .leftHalf:
+                            action = .firstThreeFourths
+                        case .lastFourth, .rightHalf:
+                            action = .lastThreeFourths
+                        case .bottomLeftEighth, .bottomRightEighth, .bottomCenterLeftEighth, .bottomCenterRightEighth:
+                            if Defaults.eightsSnapArea.userEnabled {
+                                action = .bottomCenterLeftEighth
```

Comment:
I think there might be a logic error somewhere around this? I cannot get `bottomCenterLeftEighth` (or `bottomCenterRightEighth`) to trigger.